docs/developer/extending/apps/asynchronous-webhooks.mdx

```diff
@@ -33,7 +33,9 @@ Webhook payloads are sent in POST requests.
 
 All webhooks with `targetUrl`, where the scheme is HTTP(S) will use this protocol.
 
-Saleor calculates the HMAC sha256 header - `Saleor-Signature` based on `secretKey` and the payload.
+Saleor calculates a payload signature under `Saleor-Signature`:
+ * if `secretKey` was set for a webhook - the HMAC SHA-256 header based on it and the payload
```

Comment:
I think that this note is related to async and sync webhooks.
Also, don't you think, that it would be good to add here a small note about the place where you can find a public key? Something like this: https://docs.saleor.io/docs/3.x/developer/users#rs256-public-key